src/com/epam/javatrainings/classdecomposition/PepperoniPizza.java

```diff
@@ -0,0 +1,111 @@
+package epam.javatrainings.classdecomposition;
+
+public class PepperoniPizza extends Pizza {
```

Comment:
no need to extend Pizza for each type, how you would handle the case when customer orders a custom pizza, each Pizza is an instance of Pizza class